frontend/src/ofrak/remote_resource.js

```diff
@@ -486,6 +486,21 @@ export class RemoteResource extends Resource {
       script.set(await r.json());
     });
   }
+
+  async flush_to_disk(output_file_name) {
+    await fetch(`${this.uri}/flush_to_disk`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(output_file_name),
```

Comment:
@rbs-jacob I did it this way since it felt like the natural way to do it, because it silos the act of getting the filesystem entry name to one location. I hadn't thought about using it to send a user-defined file name, but that is a good reason as well.